src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -401,6 +407,30 @@ private void AddCredentialToCache(Uri serverUri, ulong authAvail, NetworkCredent
             }
         }
 
+        private void AddResponseCookies(Uri serverUri, HttpResponseMessage response)
+        {
+            if (!_useCookie)
```

Comment:
Generally speaking, the defaults for HttpClientHandler are this (same as Desktop)

```C#
        [TestMethod]
        public void Ctor_CheckDefaults_Success()
        {
            var handler = new HttpClientHandler();
            Assert.IsTrue(handler.UseCookies);
            Assert.IsNotNull(handler.CookieContainer);
        }
```